ftest.sh

```diff
@@ -125,8 +125,10 @@ if ${TEARDOWN_ONLY:-false}; then
 fi
 
 # let's output to a dir in the tree
-rm -rf src/tests/ftest/avocado ./*_results.xml
-mkdir -p src/tests/ftest/avocado/job-results
+rm -rf install/lib/daos/TESTING/ftest/avocado ./*_results.xml
+#rm -rf src/tests/ftest/avocado ./*_results.xml
+mkdir -p install/lib/daos/TESTING/ftest/avocado/job-results
```

Comment:
This is fine for this stage of the migration but in the final result, this needs to be portable to both the `install/` prefix and the RPM `/usr` prefix.

Additionally, in the latter case, this will not be writable so this needs to be somewhere else completely in such a case.  A "tempdir" of some sort seems to be the solution.

Reply:
Tested adding TMPDIR in .build_vars.json worked for non-CI use case.

Reply:
Use TMPDIR env variable if set, otherwise, default it to PREFIX

Reply:
@GitHuaKuang I've **Unresolved** this conversation.  Generally, it is up to the person who made the original comment to resolve a conversation as they need to verify that their concern has been addressed.

As for this concern, I'm not seeing any code change which addresses the concern.  The code as it is here will fail to work when a test run from RPMs is done.

Reply:
the code is in ftest/util/server_utils.py

Reply:
This appears to be unresolved still.  In general, any hard-coded path under `install/` in this file will not work portably between RPM-based installations and non-RPM installations.  That goes for (just about) all instances of the use of `install/` in this file.